Server/db/MongoDB.js

```diff
@@ -94,6 +94,25 @@ const updateUser = async (req, res) => {
   }
 };
 
+
+/**
+ * Delete a user by ID
+ * @async
+ * @param {Express.Request} req
+ * @param {Express.Response} res
+ * @returns {Promise<UserModel>}
+ * @throws {Error}
+ */
+const deleteUser = async (req, res) => {
+  const userId = req.params.userId;
+  try {
+    const deletedUser = await UserModel.findByIdAndDelete(userId);
```

Comment:
What happens if you try to delete a user with an ID that does not exist?  This will still run, and the server will return a successful response.

If no user is deleted we need to throw an error here

Reply:
I just added the error handling their using `if statement`.
Please check to see if that's what we need.

Reply:
I think that's good, I'm not sure what delete when no user is found, but if it returns null this should be ok